src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculate = {
+    result: 0,
+
+    add(result, number) {
+      result += number;
+    },
+
+    subtract(result, number) {
+      result -= number;
+    },
+
+    multiply(result, number) {
+      result *= number;
```

Comment:
The 'multiply' method should modify the 'result' property of the 'calculate' object. Replace the local 'result' parameter with 'this.result'.